docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+
+_install_kubectl.sh_ also writes a configuration file which will embed the necessary
+authentication certificates for the particular cluster.  The configuration file is
+written to the  ${CLC_CLUSTER_HOME}/kube directory
+
+```
+export KUBECONFIG=${CLC_CLUSTER_HOME}/kube/config
+kubectl version
+kubectl cluster-info
+```
+
+### Accessing the cluster programmatically
+
+It's possible to use the locally-stored client certificates to access the api server
+```
+curl \
+   --cacert ${CLC_CLUSTER_HOME}/pki/ca.crt  \
+   --key ${CLC_CLUSTER_HOME}/pki/kubecfg.key \
+   --cert ${CLC_CLUSTER_HOME}/pki/kubecfg.crt  https://${MASTER_IP}:6443
+```
+But please note, this *does not* work out of the box with the curl binary
+distributed with OSX
+
+### Accessing the cluster with a browser
+
+We install two UIs on kubernetes. The orginal KubeUI and the newer kube
+dashboard. When you create a cluster, the script should output URLs for these
+interfaces like this:
+
+KubeUI is running at https://${MASTER_IP}:6443/api/v1/proxy/namespaces/kube-system/services/kube-ui
+kubernetes-dashboard is running at https://${MASTER_IP}:6443/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
+
+Note on Authentication to the UIs: The cluster is set up to use basic
+authentication for the user _admin_.   Hitting the url at
+https://${MASTER_IP}:6443 will require accepting the self-signed certificate
+from the apiserver, and then presenting the admin password written to file at
+
+> _${CLC_CLUSTER_HOME}/kube/admin_password.txt_
+
+
+### Configuration files
+
+Various configuration files are written into the home directory *CLC_CLUSTER_HOME* under
+_.clc_kube/${CLC_CLUSTER_NAME}_ in several subdirectories. You can use these files
+to access the cluster from machines other than where you created the cluster from.
+
+* _config/_: ansible variable files containing parameters describing the master and minion hosts
+* _hosts/_: hosts files listing access information for the ansible playbooks
+* _kube/_: kubectl configuration files, and the basic-authentication password for admin access to the kubernetes api
+* _pki/_: public key infrastructure files enabling TLS communication in the cluster
+* _ssh/_: ssh keys for root access to the hosts
+
+
+## _kubectl_ usage examples
+
+There are a great many features of _kubectl_.  Here are a few examples
+
+List existing nodes, pods, services and more, in all namespaces, or in just one
+```
+kubectl get nodes
+
+kubectl get --all-namespaces services
+
+kubectl get --namespace=kube-system replicationcontrollers
+
+```
+
+The kubernetes api server exposes services on web urls, which are protected by requiring
+client certificates.  If you run a kubectl proxy locally, kubectl will provide
+the necessary certificates and serve locally over http.
+```
+kubectl proxy -p 8001
+```
+and then access urls like http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/kube-ui/
```

Comment:
Then, you can access URLs such as `http://127.0.0.1:8001/api/v1/proxy/namespaces/kube-system/services/kube-ui/`